### PR TITLE
1.11.6 Release Notes

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/DCOS-42089

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by anithamuthu79** [comment]
@yishan-lin here is the list of issues from the Dashboard that will go through your review:
https://docs.google.com/document/d/1ZdxROJarHWsBqcSiTJFTKzI9uBMpLuo9f2Fj6O0K9JA/edit

**PS: This is a WIP item and will be complete when the issues displayed on the Dashboard have status marked as "Resolved" or "Closed".**

File: pages/1.11/release-notes/1.11.6/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+## Marathon
+- DCOS-41198 - Replace `poll_marathon_for_app_deployment()` method with `wait_for_deployments_complete()` method in dcos-test-utils.
+
+## Mesos
+- COPS-3527/DCOS_OSS-3921 - Check cache when creating Mesos resources using `make_disk_resources.py`.
+- COPS-3574/DCOS-19598/DCOS-40317/DCOS-41375 - Bump Mesos to nightly [1.5.x dd68c0b](https://github.com/mesosphere/mesos/blob/dd68c0b/CHANGELOG).
+- DCOS-24595/DCOS_OSS-3991 - Add Mesos patches for per-framework metrics.
+- DCOS-40162 - Fix container launch failures that occurred due to Mesos-bridge running out of IPs.
+
```
Comment (yishan-lin) [inline pages/1.11/release-notes/1.11.6/index.md]: Missing https://jira.mesosphere.com/browse/COPS-3371

File: pages/1.11/release-notes/1.11.6/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+## Networking
+- COPS-3279/DCOS-37703/DCOS-37703/DCOS-39703 - Fix erroneous values in service addresses stats and enable metrics forwarding.
+- COPS-3472/DCOS-38932 - Enable access to Marathon app via overlap network. 
+- COPS-3540/COPS-3517/COPS-3774/DCOS-39203/DCOS-39517/DCOS-41358 - Fix delay in overlay configuration.
+- COPS-3585/COPS-3701/DCOS-38600/DCOS-42215 - Fix deadlock issue when SSL sockets simultaneously send/receive data. 
+- COPS-3605/DCOS_OSS-3707 - Bump mesos-module to include the fix for CoreOS 1800.7.0.
+- DCOS_OSS-3697 - Fix connectivity issue between bridge and overlay networks.
+- DCOS-39707 - Fix clustering issues with `etcd`.
+
```
Comment (yishan-lin) [inline pages/1.11/release-notes/1.11.6/index.md]: Missing https://jira.mesosphere.com/browse/COPS-3576

File: pages/1.11/release-notes/1.11.6/index.md (hunk 1 above)
Comment (anithamuthu79) [reply pages/1.11/release-notes/1.11.6/index.md]: Added missing tickets and included DCOS tickets for related COPS tickets.

File: pages/1.11/release-notes/1.11.6/index.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.6/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+
+DC/OS 1.11.6 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/19d17cec3e797758e76c081efb68867d440ed4d3/CHANGELOG).
+- Marathon 1.6.549 [change log](https://github.com/mesosphere/marathon/tree/aabf74302).
+- Metronome 0.4.34 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.4).
```
Comment (orsenthil) [inline pages/1.11/release-notes/1.11.6/index.md]: Metronome Version is 0.4.4 ( the URL is correct). but the version information mentioned is incorrect.

File: pages/1.11/release-notes/1.11.6/index.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+DC/OS 1.11.6 was released on September 25, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.6/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+
+DC/OS 1.11.6 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/19d17cec3e797758e76c081efb68867d440ed4d3/CHANGELOG).
```
Comment (orsenthil) [inline pages/1.11/release-notes/1.11.6/index.md]: The Mesos URL should be - https://github.com/apache/mesos/blob/5a7ad47e8fc1a14101e47a29eb8e7e2a20d959c5/CHANGELOG

File: pages/1.11/release-notes/1.11.6/index.md (hunk 3 above)
Comment (pleia2) [inline pages/1.11/release-notes/1.11.6/index.md]: Based on the link, looks like this should just be 0.4.4 (stray "3" in there).

**Comment by anithamuthu79** [comment]
@orsenthil and @pleia2 I've made the requested changes.

**Comment by orsenthil** [comment]
LGTM, please merge it.

File: pages/1.11/release-notes/1.11.6/index.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+
+DC/OS 1.11.6 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/5a7ad47e8fc1a14101e47a29eb8e7e2a20d959c5/CHANGELOG).
+- Marathon 1.6.549 [change log](https://github.com/mesosphere/marathon/tree/aabf74302).
+- Metronome 0.4.4 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.4).
+
+# Notable Changes in DC/OS 1.11.6
+- DCOS-22310/DCOS-40495/DCOS-41282 - Update DC/OS UI for [1.11+v1.23.0](https://github.com/dcos/dcos-ui/blob/1.11+v1.23.0/CHANGELOG.md). [oss type="inline" size="small" /]
+- DCOS_OSS-4106/DCOS_OSS-4109 - Bump ZooKeeper to [3.4.13](https://zookeeper.apache.org/doc/r3.4.13/releasenotes.html). [enterprise type="inline" size="small" /]
```
Comment (jgehrcke) [inline pages/1.11/release-notes/1.11.6/index.md]: Enterprise label is wrong

**Pull request merged**